plottr/plot/mpl.py

```diff
@@ -912,7 +949,12 @@ def _plotData(self, adjustSize: bool = False) -> None:
         if not self.dataIsComplex():
             self.complexRepresentation = ComplexRepresentation.real
         else:
-            self.complexRepresentation = self.complexPreference
+            if ComplexRepresentation.realAndImag:
+                self.complexRepresentation is ComplexRepresentation.realAndImag
+            if ComplexRepresentation.real:
+                self.complexRepresentation is ComplexRepresentation.real
+            if ComplexRepresentation.magAndPhase:
+                self.complexRepresentation is ComplexRepresentation.magAndPhase
```

Comment:
This code does nothing.  The if statements does not compare against anything so they will always evaluate the same way. `is` does not assign to anything that only checks if one object is the same as the other one and return a bool which the code just discards. 

I suggest that you revert back to the original logic

Reply:
I completely removed these conditional statements.